src/patient-registration/input/custom-input/autosuggest.css

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
We can make a separate ticket to fix the autosuggest styles once we have merged this ticket into a new branch.

Reply:
Cool...Will the changes go to the style guide module?

Reply:
I'm not too sure as there is currently discussion regarding using a different UI style guide (e.g. MaterialUI), so this may change. I think, at the moment, we should focus on functionality/clean code/refactoring and then tackle the styles issue